terms-conditions/participation-form.md

```diff
@@ -0,0 +1,94 @@
+### Certified FDC3 Conformance Program
+#### Participation Form
+
+Complete this form for each Participant (company or other entity) that desires to participate in the Certified FDC3 Conformance Program and to use the Certified FDC3 Marks or Participant FDC3 Combinations. FDC3 and the Certified FDC3 Marks are trademarks of The Linux Foundation. Capitalized terms used herein and not otherwise defined shall have the same meanings set forth in the Program Terms.
+
+By signing below and submitting this form to The Linux Foundation by emailing a scan to [help@finos.org](mailto:help@finos.org)):
+
+1.  The Participant agrees to the Terms and Conditions of the Certified FDC3 Conformance Program (the "**Program Terms**"), available at <https://github.com/finos/fdc3-conformance-testing/tree/master/terms-conditions/FDC3-Certified-Terms.pdf>.
+2.  The Participant confirms that the products and services identified below as Qualifying Offerings have passed all of the self-tests described in the Certification Guide, and are Qualifying Offerings under the Program Terms.
+3.  The Participant confirms that it has submitted to FINOS the results of the self-tests prior to its first public use of the Certified FDC3 Marks associated with the corresponding version of FDC3.
+4.  The Participant confirms that it will either (a) maintain conformance of the Qualifying Offerings with later versions of FDC3, or (b) cease use of the Certified FDC3 Marks and Participant FDC3 Combinations at the end of the applicable conformance time period described in the Program Terms.
+5. The Participant confirms that it has listed below all Participant FDC3 Combinations that it intends to use with the Qualifying Offerings.
+6.  The Participant confirms that it will promptly submit an updated Participant Form to The Linux Foundation prior to (a) using the Certified FDC3 Marks with Qualifying Offerings not listed here, or (b) using Participant FDC3 Combinations not listed here.
+7.  I confirm that I am authorized to make the above statements and to submit this form on behalf of the Participant.
+
+
+#### Participant Information
```

Comment:
Why is this all manual?

@robmoffat have you checked with @grizzwolf pro/cons as to whether this should be a form in our Hubspot rather than a manual print / scan thing?

More as internal details, maybe not worth for the volume, but we could have a docusign set up rather than scanning / printing and managing a manual process.

Maybe let's chat on this internall first?

Reply:
ok

Reply:
as discussed - if there were more traffic or we had more time, I think a form would make a lot of sense.  It's going to be 4 legal officers signing 4 pieces of paper and mailing them to us though which I think means we should stick with quick & dirty.

Reply:
This is now automated on docusign and @robmoffat gets a notification.